forms-flow-api/src/formsflow_api/models/form_process_mapper.py

```diff
@@ -51,6 +51,9 @@ class FormProcessMapper(AuditDateTimeMixin, AuditUserMixin, BaseModel, db.Model)
     deleted = db.Column(db.Boolean, nullable=True, default=False)
     task_variable = db.Column(JSON, nullable=True)
     version = db.Column(db.Integer, nullable=False, default=1)
+    can_bundle = db.Column(db.Boolean, nullable=False, default=False)
+    is_bundle = db.Column(db.Boolean, nullable=False, default=False)
+    description = db.Column(db.String(300), nullable=True)
```

Comment:
May be remove the length ~300, and just leave it as String as description could be large for some clients.